server/channels/app/platform/web_hub_test.go

```diff
@@ -237,10 +239,14 @@ func TestHubConnIndex(t *testing.T) {
 				wc4.SetConnectionID(model.NewId())
 				wc4.SetSession(&model.Session{})
 
-				connIndex.Add(wc1)
-				connIndex.Add(wc2)
-				connIndex.Add(wc3)
-				connIndex.Add(wc4)
+				errAdd := connIndex.Add(wc1)
```

Comment:
nit: we can simply do `require.NoError(t, connIndex.Add(wc1))`

Reply:
2/5 I find it harder to read if both the test logic and the error check are in the same line.

Reply:
Yeah, I like those on separate lines even if it's a bit wordier. I think we sometimes test pure functions by calling them inside of things like `assert.Equal`, but I wouldn't expect to see something with a side effect inside a `require.NoError` for some reason

Reply:
> 2/5 I find it harder to read if both the test logic and the error check are in the same line.

That is interesting. I don't find it harder at all. But no biggie :)